examples/linear-regression/src/main/scala/org/apache/spark/examples/ml/LinearRegressionExample.scala

```diff
@@ -108,8 +110,16 @@ object LinearRegressionExample {
     println(s"LinearRegressionExample with parameters:\n$params")
 
     val training = spark.read.format("libsvm")
-      .load(params.input).toDF("label", "features")
+      .load(params.input).toDF("label", "features").repartition(4)
```

Comment:
don't use repartition in user code, you should check if partition is enough and change in oap mllib code.